src/admin.js

```diff
@@ -447,6 +447,16 @@ export class StitchAdminClient extends StitchClient {
               post: (data) => api._post(`${graphqlUrl}`, data),
               validate: () => api._get(`${graphqlUrl}/validate`)
             };
+          },
+
+          realm: () => {
+            const realmUrl = `${appUrl}/realm`;
+            return {
+              config: () => ({
+                get: () => api._get(`${realmUrl}/config`),
+                update: (data) => api._put(realmUrl, { body: JSON.stringify(data) })
```

Comment:
Oh wait actually, both of these hit the `/config` endpoint, right? So it should also be `${realmUrl}/config`. Or, even better, we could just revise the `const` to be
```
realmConfigUrl = `${appUrl}/realm/config`
```
and then reference that in `get` and `update`